AssignmentFive/AssignmentFive/AssignmentFive/ViewControllers/InitialViewController.m

```diff
@@ -0,0 +1,78 @@
+
+
+#import "InitialViewController.h"
+#import "AddFriendViewController.h"
+#import "CustomCellTableViewCell.h"
+#import "AppDelegate.h"
+#import "Person.h"
+
+@interface InitialViewController (){
+    NSMutableArray *tempArray;
+}
+
+@end
+
+@implementation InitialViewController
+@synthesize tableViewListOfFriends;
+- (void)viewDidLoad {
+    [super viewDidLoad];
+  
+    
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+   
+}
+
+-(void)viewWillAppear:(BOOL)animated{
+    
+    [super viewWillAppear:animated];
+    
+    tempArray=[((AppDelegate *)[UIApplication sharedApplication].delegate).arrayOfFriends mutableCopy ];
+    [tableViewListOfFriends reloadData];
+    NSLog(@"in view will appear");
+}
+
+#pragma mark :- button action
+- (IBAction)actionButtonHandler:(UIButton *)sender {
```

Comment:
Try to use button action definitions as buttonAddFriendTapped.
We are following this practice.